src/app/content/components/SearchResultsSidebar/styled.tsx

```diff
@@ -0,0 +1,210 @@
+
+import styled, { css } from 'styled-components/macro';
+import { Search } from 'styled-icons/fa-solid/Search';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import { iconSize } from '../../../components/Details';
+import { navDesktopHeight } from '../../../components/NavBar';
+import { labelStyle, textRegularLineHeight, textRegularStyle } from '../../../components/Typography';
+import theme from '../../../theme';
+import {
+    bookBannerDesktopMiniHeight,
+    searchResultsBarDesktopWidth,
+    toolbarDesktopHeight,
+    toolbarMobileElementsHeight,
+  } from '../constants';
+import ContentLinkComponent from '../ContentLink';
+import { Summary } from '../Sidebar/styled';
+import { toolbarIconStyles } from '../Toolbar';
+
+const searchResultsBarVariables = {
+    backgroundColor: '#f1f1f1',
+    iconRightPadding: 0.7,
+    mainPaddingDesktop: 3,
+    mainPaddingMobile: 2,
```

Comment:
is this supposed to be the same as the page padding in the app theme?

Reply:
probably, but I followed the invision design, says 30px, already removed the mainPaddingMobile that was unused.